src/test/java/org/folio/rest/impl/InvoiceLinesApiTest.java

```diff
@@ -280,6 +286,37 @@ public void deleteInvoiceLinesBadLanguageTest() {
     verifyDeleteResponse(endpoint, TEXT_PLAIN, 400);
   }
 
+  @Test
+  public void fundValidationTest()
+  {
+    ValidateFundDistributionsRequest reqData = getMockAsJson(FUND_VALIDATOR_MOCK_DATA_PATH).mapTo(ValidateFundDistributionsRequest.class);
+     verifyPut(INVOICE_LINE_FUNDS_VALIDATOR_ID_PATH, reqData, "", 204);
+  }
+  @Test
+  public void fundValidationDupAdjIdTest()
+  {
+    ValidateFundDistributionsRequest reqData = getMockAsJson(FUND_VALIDATOR_MOCK_DATA_PATH_DUP_ADJ_ID).mapTo(ValidateFundDistributionsRequest.class);
+     verifyPut(INVOICE_LINE_FUNDS_VALIDATOR_ID_PATH, reqData, "", 400);
```

Comment:
You also need to check remainingAmount field in your test in some way, not only response codes, because this field is very important to test calculation logic

Reply:
sure , updated test